draft-ietf-httpbis-origin-frame.md

```diff
@@ -153,6 +153,16 @@ The Origin Set is also affected by the 421 (Misdirected Request) response status
 MUST create the ASCII serialisation of the corresponding request's origin (as per {{!RFC6454}},
 Section 6.2) and remove it from the connection's Origin Set, if present.
 
+Note:
+
+: A server that acts as an alternative service {{?RFC7838}} might remove the origin that a
+  client creates the connection for by sending an ORIGIN frame.  The single value that is in the
+  newly-initialized Origin Set will contain the port of the alternative service rather than that
+  of the origin.  For instance, a client making requests for "https://example.com" that is
+  directed to an alternative at ("h2", "x.example.net", "8443") will seed the origin set with
+  "https://example.com:8443".  This problem can be avoided by explicitly including an origin
+  with the correct port in the ORIGIN frame.
+
```

Comment:
I thought that the example was illustrative: if you Alt-Svc with a different port, then send ORIGIN you get in a bind unless you include the original origin in the ORIGIN frame because the implicitly-included origin uses the alternative service port.

I've been trying to find a word for a. the origin that is seed for the origin set, and b. the origin that the client is chasing with alternative services.  I don't think that "original origin" is very helpful.